src/components/PageHeader.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <header>
+        <div class="flex flex-cols justify-between px-4 pt-7 items-center">    
+            <h1 class="text-2xl lg:text-7xl font-semibold">To do List</h1>
+            <button @click="showForm" class="lg:w-[59.64px] lg:h-[59.64px] w-8 h-8 bg-green text-xl text-white lg:text-4xl rounded-full col-end-auto">+</button>
```

Comment:
please refer to style guide, and order the props and events based on that, events are the last attribute based on order of importance, since you want to know what goes into the component first, before seeing what will come back from it.